Cargo.toml

```diff
@@ -2108,3 +2289,7 @@ codegen-units = 1
 inherits = "release"
 lto = "fat"
 panic = "abort"
+
+[package.metadata.docs.rs]
+all-features = true
```

Comment:
This one seems also unrelated. I don't remember, does docs.rs shows the warning about feature automatically or does that needs to be added manually?

Reply:
True, not related. I saw this in another project and added it as a precaution in a separate commit. Should I skip it?

Reply:
I removed it just to avoid unexpected outcomes. We can add it in another PR if we consider the change and decide we like it.